packages/components/src/custom-select-control/index.js

```diff
@@ -67,13 +67,11 @@ export default function CustomSelectControl( {
 		getItemProps,
 		isOpen,
 		highlightedIndex,
-		selectedItem,
 	} = useSelect( {
 		initialSelectedItem: items[ 0 ],
 		items,
 		itemToString,
 		onSelectedItemChange,
-		selectedItem: _selectedItem,
 		stateReducer,
 	} );
 
```

Comment:
Aren't there other `a11y` props that need to be set on the selected item? This won't let the hook do it for the initial state when using this in a controlled manner. It will think `items[ 0 ]` is always the selected item.

Reply:
Thanks for spotting that! I've fixed it so that all the attributes are in the correct places again, and all keyboard functionality is restored. The one thing I'm not super happy with is that, if a new value is set elsewhere, immediately reopening the dropdown will show the previous value incorrectly highlighted. I couldn't find a good workaround for that, but it seems to me less of an issue than what this PR is trying to fix 😅 
Ultimately I think the best solution would be for Downshift to introduce an option to disable its `aria-live` announcements altogether, so we can use our custom ones and avoid clashes with other components (cc @silviuaavram would be great to have your opinion on this). Otherwise, if we find ourselves having to reimplement a ton of logic on our side, it might be easier to build our own. Happy to hear other ideas!